src/Event/ReadModel/JSONLD/EventLDProjector.php

```diff
@@ -362,9 +364,17 @@ protected function applyEventCopied(
         $eventJsonLD->{'@id'} = $this->iriGenerator->iri($eventCopied->getItemId());
 
         // Set the new calendar.
+        $calendarJsonLD = $eventCopied->getCalendar()->toJsonLd();
+        // workaround - if user removes week scheme values, we need to explicitly
+        // set the field to [] here so the original value
+        // will get overridden in the merge step
+        if ($this->isPeriodicCalendarWithoutWeekScheme($eventCopied->getCalendar())) {
+            $calendarJsonLD['openingHours'] = [];
+        }
```

Comment:
Nit: Might be better to `unset()` the property in this case (after the `array_merge`), but this might work as well.

Reply:
I just checked the search API and it checks if the property is set or not to index the datetime-ranges the event is available, so I think we should definitely `unset()` it instead. https://github.com/cultuurnet/udb3-search-service/blob/master/src/ElasticSearch/Offer/AbstractOfferJsonDocumentTransformer.php#L125